megazords/full/android/src/main/java/mozilla/appservices/Megazord.kt

```diff
@@ -5,13 +5,15 @@
 package mozilla.appservices
 
 import org.mozilla.appservices.full_megazord.BuildConfig
+import org.mozilla.appservices.full_megazord.ensureNssInitialized
 
 class Megazord {
     companion object {
         @JvmStatic
         fun init() {
             System.setProperty("mozilla.appservices.megazord.library", "megazord")
             System.setProperty("mozilla.appservices.megazord.version", BuildConfig.LIBRARY_VERSION)
+            ensureNssInitialized()
```

Comment:
In the future, maybe we could initialize viaduct and other dependencies here too.